lib/raven/integrations/rack.rb

```diff
@@ -67,6 +67,8 @@ def call(env)
   end
 
   module RackInterface
+    REQUEST_ID_HEADERS = Set.new(%w(HTTP_X_REQUEST_ID action_dispatch.request_id)).freeze
```

Comment:
If this is not mutable to users, why does it need to be a set instead of just an array?

Reply:
oh~ I was thinking of a faster `.include?` through `Set`, which could be faster with O(1) instead of O(n), but I think only two elements I guess it's fine here, change back to array